kotlin-eclipse-ui/src/org/jetbrains/kotlin/ui/editors/Configuration.java

```diff
@@ -185,12 +201,23 @@ public IRegion getSubject(ITextViewer textViewer, int offset) {
         
         @Override
         public String getInformation(ITextViewer textViewer, IRegion subject) {
-            return getInformation2(textViewer, subject).toString();
+            Object result = getInformation2(textViewer, subject);
+            if (result != null) {
+                return result.toString();
+            } else {
+                return null;
+            }
```

Comment:
Minor remark: to reduce nested operators you can return value not in the `else` block or you can use ternary operator here.